tools/build/python_config.py

```diff
@@ -111,7 +111,7 @@ def normpath(path: str) -> str:
 
 	config_vars.link_libs = _stable_unique([
 			*itertools.chain(*(
-				(v.removeprefix('-l') for v in value.split())
+				(v.removeprefix('-l') for v in value.split() if v.startswith('-l'))
```

Comment:
This concerns me, but I'm not completely sure why...

Reply:
Open to alternatives :) Either way my LIBS and SYSLIBS contain entries that aren't meant to be linked libs, as explained in the PR text.

Reply:
I wonder what these contain on other platforms, its too long ago for me to remember why this line is this the way it is, but there could have been a reason

Reply:
I think it may be good to compile on Windows and Linux to see what the values in `'LIBS' and 'SYSLIBS' are and whether all should be added to `config_vars.link_libs`. That's probably the only way to confirm for sure. I don't have convenient access to either platform though.

Reply:
When I have time, I can test this on Linux with `gcc` and `mingw`, but not an actual Windows system. I think that will be sufficient tho.

Reply:
I remember that scons uses linkflags for .lib and LIBS for liblibrary.

Reply:
For the record, on macOS `print([[v for v in value.split()] for name in ('LIBS', 'SYSLIBS') if (value := sysconfig_vars.get(name))])` prints `[['-ldl', '-framework', 'CoreFoundation']]`. `ld` is a library (albeit a duplicate entry for one) but `-framework CoreFoundation` is one argument split in two words and therefore these are not library entries.

Reply:
Pythons own config is to inconsistent to really learn anything from, and everything here seems to be building just fine, so will just call this a non issue.